paddle/operators/conv_shift_op.cu

```diff
@@ -33,9 +33,9 @@ inline int div_up(int x, int y) { return (x + y - 1) / y; }
 // y is fairly small. For large y, it would probably be more efficient
 // to also tile across y.
 template <typename T>
-__global__ void conv_shift_forward(const T *x, const T *y, T *out, int x_width,
-                                   int y_width, int y_half_width,
-                                   int batch_size) {
+__global__ void ConvShiftForward(const T *x, const T *y, T *out, int x_width,
+                                 int y_width, int y_half_width,
+                                 int batch_size) {
   extern __shared__ T mem[];
 
   int tx = threadIdx.x;
```

Comment:
Although the problem mentioned in issue has been solved, there is a problem with the ConvShiftForward function.
Please recheck line 62~68, you should not use `return` before `__syncthreads();`

Reply:
Good catch. Let me fix this now quickly.